tests/tests/test_unit/test_plugin/test_register.py

```diff
@@ -0,0 +1,13 @@
+from brainreg.napari.register import get_available_atlases
+
+atlas_name = "allen_mouse_50um"
+
+
+def test_get_available_atlases():
+    atlases = get_available_atlases()
+
+    # arbitrary selection of atlases
+    assert float(atlases["allen_mouse_10um"]) >= 0.3
+    assert float(atlases["allen_mouse_25um"]) >= 0.3
+    assert float(atlases["allen_mouse_50um"]) >= 0.3
+    assert float(atlases["mpin_zfish_1um"]) >= 0.4
```

Comment:
What's the purpose of this comparison here? I don't understand why casting an atlas to a float should result in something >=0.3.

If we're just checking for the presence of certain atlases, we could do something like

```suggestion
    expected_atlases = [
        "allen_mouse_10um",
        "allen_mouse_25um",
        "allen_mouse_50um",
        "mpin_zfish_1um",
    ]
    
    for a in expected_atlases:
        assert a in atlases.keys(), f"{a} is not in the list of expected atlases"
```

Also in general I prefer this for-loop style for tests like this where we have a bunch of stuff that we're testing in the same way, but that's personal preference. Just means that we only have to update the items in a list when we expect more atlases.

Reply:
This code used to be in `brainglobe-segmentation`, and TBH I just copied over the test without looking at what it did! Your version is much better.